.github/workflows/CI.yml

```diff
@@ -48,6 +48,8 @@ jobs:
           # For Codecov, we must also fetch the parent of the HEAD commit to
           # be able to properly deal with PRs / merges
           fetch-depth: 2
+      - name: "Install extra dependencies"
+        run: sudo apt-get install expect
```

Comment:
Gotta adapt this for macos 😆

Reply:
Is `expect` pre-installed on the macOS runners?

Reply:
Yes, in fact it ships with macOS (resp. with Xcode)